centos_ci.sh

```diff
@@ -292,10 +289,9 @@ function perform_release() {
   curl http://minibot.19cf262c.svc.dockerapp.io:9009/hubot/centosci -H "Content-Type: application/json" -d '{"payload":{"status":"success","message":'"\"$MESSAGE\""',"url":'"\"$URL\""'}}' || true
 }
 
+# Accepts two arguments: repo owner and location in the artifact server
```

Comment:
Could you please document in the function docs which parameters a function expects and what they mean. See also https://google.github.io/styleguide/shell.xml#Comments

It is really hard to figure out what's going on if one just sees $1, $2, etc

Reply:
Thanks. Make sense :+1: